lib/engine/game/g_1856.rb

```diff
@@ -309,6 +311,33 @@ def cert_limit
         POST_NATIONALIZATION_CERT_LIMIT[num_corporations][@players.size]
       end
 
+      def destination_check
+        @corporations.each do |corp|
+          # The capitalization method of unparred corporations is nil
+          next unless corp.capitalization == :escrow
+
+          destinated!(corp) if hexes_connected?(@destinations[corp.id].first, @destinations[corp.id].last)
+        end
+      end
+
+      def hexes_connected?(start_hex_id, goal_hex_ids)
+        tokens = hex_by_id(start_hex_id).tile.cities.map { |city| [city, true] }.to_h
```

Comment:
why is this a hash? doesn't seem like you use it excepts as an array?

Reply:
It's because `node.walk` needs `visited` to be a hash